src/options_app.c

```diff
@@ -0,0 +1,1479 @@
+#include "global.h"
+#include "bg_window.h"
+#include "data/resdat.naix"
+#include "font.h"
+#include "gf_gfx_loader.h"
+#include "msgdata/msg/msg_0045.h"
+#include "msgdata.h"
+#include "msgdata/msg.naix"
+#include "options_app.h"
+#include "options.h"
+#include "unk_02005D10.h"
+#include "unk_0200CF18.h"
+#include "unk_0200FA24.h"
+#include "unk_020183F0.h"
+#include "unk_02023694.h"
+#include "render_text.h"
+#include "render_window.h"
+#include "sound.h"
+#include "system.h"
+#include "touchscreen.h"
+#include "vram_transfer_manager.h"
+
+// FIXME: Import this from field/launch_application when #290 is merged
```

Comment:
disagree, this should be in the options_app.h, and should be included in launch_application